hibernate-core/hibernate-core.gradle

```diff
@@ -61,6 +61,8 @@ dependencies {
         transitive = true
     }
     testImplementation "joda-time:joda-time:2.3"
+    testImplementation testLibs.archunitJUnit5
```

Comment:
I'm using ArchUnit to be able to assert annotations with retention policy `RetentionPolicy.CLASS`.